core/templates/dev/head/pages/exploration_editor/preview_tab/PreviewTab.js

```diff
@@ -112,7 +114,8 @@ oppia.controller('PreviewTab', [
     };
 
     $scope.loadPreviewState = function(stateName, manualParamChanges) {
-      oppiaPlayerService.initSettingsFromEditor(stateName, manualParamChanges);
+      ExplorationPlayerService.initSettingsFromEditor(stateName, 
+						      manualParamChanges);
```

Comment:
Could you bring stateName in the next line too and no need to move the parameters under the parenthesis, just indent it once, that would be enough! For example, this is what I mean by indent it once : 

![indentation](https://user-images.githubusercontent.com/7807557/30769980-041eed96-a043-11e7-8f17-1be090412945.jpeg)